DoorGate/ViewController.swift

```diff
@@ -10,48 +10,57 @@ import UIKit
 import RxSwift
 import RxCocoa
 
-let stateString = ["Open", "Occupied", "Locked"]
+let countString = ["Open", "Occupied", "Locked"]
```

Comment:
This should probably go inside the `DoorState` enum above maybe as a convenience dynamic var: `var asString: String { ...`